components/mintKamon/GenerateImageForm.tsx

```diff
@@ -1,13 +1,96 @@
-import { Button, Heading } from "@chakra-ui/react"
-import { Layout } from "@/components/layouts/layout"
+import { useAccount } from 'wagmi'
+import { useState } from 'react'
+import { Button, Heading, Text, FormControl, FormLabel, Input } from '@chakra-ui/react'
+import { Layout } from '@/components/layouts/layout'
 
 const GenerateImageForm = () => {
+  const ipfsApiEndpoint = 'https://api.staging.sakazuki.xyz/henkaku/ipfs'
+  const { data } = useAccount()
+  const [input, setInput] = useState('')
+  const [disabled, setDisabled] = useState(false);
+  const handleInputChange = (e: any) => setInput(e.target.value)
```

Comment:
［IMO］In general, Any should be used as little as possible, but it may be difficult if you're not used to it, so you can ignore. some IDEs will show you the type, so you can rely on that.

Reply:
Agree. Just trying to go fast, sorry! 😭

I did a quick `any` pass. Thank you. Will push later.